ts-client/README.md

```diff
@@ -0,0 +1,77 @@
+# Meteora Pool Farm SDK
+
+<p align="center">
+<img align="center" src="https://vaults.mercurial.finance/icons/logo.svg" width="180" height="180" />
+</p>
+<br>
+
+## Getting started
+
+NPM: https://www.npmjs.com/package/@mercurial-finance/farming-sdk
+
+SDK: https://github.com/MeteoraAg/reward-pool
+
+<!-- Docs: https://docs.mercurial.finance/mercurial-dynamic-yield-infra/ -->
+
+Discord: https://discord.com/channels/841152225564950528/864859354335412224
+
+<hr>
+
+## Install
+
+1. Install deps
+
+```
+npm i @mercurial-finance/farming-sdk @project-serum/anchor @solana/web3.js @solana/spl-token @solana/spl-token-registry
+```
+
+2. Initialize AmmImpl instance
+
+```ts
+import PoolFarmImpl from "@mercurial-finance/farming-sdk";
+import { PublicKey } from "@solana/web3.js";
+import { Wallet, AnchorProvider } from "@project-serum/anchor";
+
+// Connection, Wallet, and AnchorProvider to interact with the network
+const mainnetConnection = new Connection("https://api.mainnet-beta.solana.com");
+const mockWallet = new Wallet(new Keypair());
+const provider = new AnchorProvider(mainnetConnection, mockWallet, {
+  commitment: "confirmed",
+});
+// Alternatively, to use Solana Wallet Adapter
+
+const farmingPool = await PoolFarmImpl.getFarmAddressByPoolAddress(
```

Comment:
it should be farmingPools 
And you should note that a pool address may have more farming pools

Reply:
Updated